deploy/olm-catalog/kogito-operator/0.8.0/kogito-operator.v0.8.0.clusterserviceversion.yaml

```diff
@@ -27,17 +27,15 @@ metadata:
             "name": "kogito-data-index"
           },
           "spec": {
-            "cpuLimit": "",
-            "cpuRequest": "",
-            "image": "quay.io/kiegroup/kogito-data-index:latest",
+            "image": {
+              "tag": "0.7.0"
```

Comment:
Shouldn't be here 0.8.0 or latest?

Reply:
See my comment below: https://github.com/kiegroup/kogito-cloud-operator/pull/184#discussion_r386450220